app/controllers/application_controller.rb

```diff
@@ -24,16 +30,48 @@ def current_user
       end
     end
     
+  helper_method :admin_access_html
+    def admin_access_html
+      if is_admin_html || is_instructor_html
+        true
+      else
+        false
+      end
+    end
+    
+  helper_method :admin_access
+    def admin_access
```

Comment:
Should it really be called admin_access is its for both admin and instructors? Something like organizer_access, super_access, elevated_access, privileged_access, admin_instructor_access?